RefactoringsExperiments/src/main/java/org/jetbrains/research/extractMethodExperiments/metrics/MetricCalculator.java

```diff
@@ -30,28 +31,38 @@
 import static org.jetbrains.research.extractMethodExperiments.utils.StatementListUtil.ListToStr;
 
 public class MetricCalculator {
-    private static final Logger LOG = Logger.getInstance(MetricCalculator.class);
+    private static final Logger LOG = LogManager.getLogger(MetricCalculator.class);
     private final String statementsStr;
     private final PsiMethod method;
+    private final String repoPath;
+    private final String filePath;
     private final int beginLine;
     private final int endLine;
     private final FeaturesVector featuresVector;
 
-    public MetricCalculator(List<PsiStatement> statements, PsiMethod method, int beginLine, int endLine) {
-        this.statementsStr = ListToStr(statements);
+    public MetricCalculator(String code, PsiMethod method, int beginLine, int endLine) {
```

Comment:
do we need two constructors? we've already discussed it and agreed that there is no need for both of them. something changed?